modules/analyze.py

```diff
@@ -200,15 +200,17 @@ def bulk_analyze_text(file_list, model_list, issue_list, results_file, summarize
 
     """
 
+    results_file = os.path.join(output_dir, "analyze_results.xlsx")
```

Comment:
This also breaks a little bit of a cardinal rule of code collaboration  in that by removing results_file as a input variable and replacing it with a similar string input output_dir, the change is not backwards compatible with the pre-existing notebooks. Better to add outpur_dir as a keyword argument (output_dir='your_output_path'), because new keyword arguments are backwards compatible

Reply:
Good suggestion for the backwards compatibility! 
The reason I put a static file name here was because the name of output_dir will always be different when called from the run.py script, so it will never be overwritten. But you're right, if we call this function directly using the same output_dir, the file will be overwritten. 
I will update this.